media-plugins/live/live-2020.03.06.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Yep, I was going for minimal changes at first because I wasn't sure about improving the ebuild as a whole, but I figured let's do it for the exercise.